src/main/java/org/opentripplanner/middleware/utils/JsonUtils.java

```diff
@@ -61,6 +63,7 @@ public static void logMessageAndHalt(Request request, int statusCode, String mes
             return mapper.readValue(json, clazz);
         } catch (JsonProcessingException e) {
             LOG.error("Unable to get POJO from json for " + clazz.getSimpleName(), e);
+            // TODO: Bugsnag?
         }
         return null;
     }
```

Comment:
Most of these methods in here should not return null and instead should throw the exception. This will avoid NPEs elsewhere.

Reply:
Again, I think this is out of scope for this MVP PR. I will add to #51.